apache/superset
# fix(maps): Move Overseas department and regions closer to France mainland

feat(maps): add drom closer to mainland France

### SUMMARY
The notebook repositions the overseas departments and regions of France (DROM) closer to mainland.
The file france.geojson is generated from the notebook.



### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before (only France metropolitan departments): 
<img width="850" alt="france_before" src="https://github.com/apache/superset/assets/9881262/97b1e707-c86b-4d7c-8027-23cf84f2448a">

After (DROM included):
<img width="851" alt="france_after" src="https://github.com/apache/superset/assets/9881262/88a0fbfe-9818-4467-adfd-ac2c6013264a">


### TESTING INSTRUCTIONS

Execute the notebook to observe that France DROM are effectively moved closer to mainland so that it can be display in a single map.

### ADDITIONAL INFORMATION
